Gruntfile.js

```diff
@@ -80,6 +81,33 @@ module.exports = function(grunt) {
         finRepl.startRepl();
     });
 
+    grunt.registerTask('kill-processes', function() {
+
+          // looks up the process by the program name
+          ps.lookup({
+              command: 'openfin.exe'
+          }, function(err, processList) {
+              if (err) {
+
+                  // if error occurs then thow error
+                  throw new Error( err );
+              }
+
+              if ( processList.length !== 0 ) {
+
+                  // return the process ids of each process found
+                  processList = processList.map( i => {
+
+                        return pid: i.pid
+                  }).forEach( process => {
+
+                      // kill each process
+                      ps.kill(process.pid)
+                  })
+              }
```

Comment:
Less code: `processList.forEach(process => ps.kill(process.pid));`
No need for `if (processList.length !== 0) {...}` check and `.map` in this case.